packages/reset.css

```diff
@@ -0,0 +1,96 @@
+/* Box sizing rules */
```

Comment:
packages/reset.css is not the correct place to keep this file 

For the time being it might be better to copy the `preview-head.html` to `web-components` and when we decide how to handle css reset we can extract or remove these files

Reply:
Added css reset styles to web components' storybook until clarifying how to handle it globally.

Reply:
This works for me—I’m wondering if we’ll end up with enough utility scripts to merit a `utils` npm package that a reset could be a part of. For instance, the online/offline detection we’ve been talking about. I’ll refactor a reset into that package when/if necessary!